grails-app/controllers/au/org/ala/biocollect/merit/ImageController.groovy

```diff
@@ -134,6 +155,56 @@ class ImageController {
      * Uploads image file or survey method supporting file to staging area
      * @return file metadata for rending in view
      */
+    @Operation(
+        method = "POST",
+        tags = "biocollect",
+        operationId = "uploadimage",
+        summary = "Stage an image or document to BioCollect server.",
+        requestBody = @RequestBody(
+                description = "File to upload",
+                content = @Content(
+                        mediaType = "multipart/form-data",
+                        schema = @Schema(
+                                implementation = FileUpload.class
+                        )
+                )
+        ),
+        parameters = [
+            @Parameter(
+                    name = "role",
+                    in = ParameterIn.QUERY,
+                    description = "Role of the document.",
+                    schema = @Schema(
+                            name = "role",
+                            type = "string",
+                            allowableValues = [
+                                    "banner", "blogImage", "image", "information", "logo",
+                                    "mainImage", "methodDoc", "other", "photo", "photoPoint", "primary",
+                                    "programmeLogic", "projectHighlightReport", "projectPlan", "projectVariation",
+                                    "rssFeed", "stageReport", "surveyImage"
+                            ]
+                    )
+            )
+        ],
+        responses = [
+                @ApiResponse(
+                        description = "Images upload result or error message.",
+                        responseCode = "200",
+                        content = @Content(
+                                mediaType = "application/json",
+                                schema =  @Schema(
+                                        oneOf = [
+                                                UploadResponse.class,
+                                                FileUploadErrorResponse.class
+                                        ]
+                                )
+                        )
```

Comment:
Can you please add CORS headers section as per https://confluence.csiro.au/display/ALASD/OIDC+Migration+on+ALA+Applications?